README.md

```diff
@@ -17,121 +17,5 @@ some may not be fully compatible as the version available in NixOS or
 [emacs-overlay](https://github.com/nix-community/emacs-overlay) may not be
 compatible with the `doom-emacs` requirements.
 
-## Getting started
```

Comment:
After #267, I think it would be nice to have a quick start section:

```
$ nix run github:nix-community/nix-doom-emacs
```

And then redirect the user to the docs.